app/views/page/show.html.erb

```diff
@@ -116,21 +116,21 @@
             <% end %>
           </div>
 
-        <%# Subpage Hyperlink %>
+          <%# Subpage Hyperlink %>
         <% when 'PageSubpageHyperlinkComponent' %>
-          <div class="grid-row margin-bottom-3  grid-col-8 <%= 'margin-x-auto' if @page.narrow? %>">
-            <a href="<%= component.url %>" class="width-100 dm-internal-link">
-              <h2 class="font-sans-xl display-inline x0-top"><%= component.title %></h2>
+          <div class="grid-row margin-bottom-5 grid-col-12 desktop:grid-col-8 <%= 'margin-x-auto' if @page.narrow? %>">
+            <a href="<%= component.url %>" class="width-100 dm-internal-link subpage-hyperlink-component">
+              <h2 class="font-sans-xl display-inline x0-top text-underline"><%= component.title %></h2>
```

Comment:
can we remove the `text-underline` here and then get rid of the `subpage-hyperlink-component` class?

Reply:
If I do that, then the extra line shows up at the end of the text again. 
![Screen Shot 2021-06-14 at 1 19 57 PM](https://user-images.githubusercontent.com/34111449/121932795-4b84d000-cd13-11eb-8dcc-8ab68e2b9c4b.png)
![Screen Shot 2021-06-14 at 1 20 07 PM](https://user-images.githubusercontent.com/34111449/121932804-4e7fc080-cd13-11eb-9c88-42316014ccd8.png)

Reply:
Oh I didn't see that. Good call. You can ignore this then :)

Reply:
All right, sounds good :blush: